.github/workflows/createTagReleases.yml

```
@@ -20,17 +20,29 @@ on:
 jobs:
   # This workflow contains a single job called "greet"
   createRelease:
+      name: createRelease
       runs-on: ubuntu-latest
       strategy:
         matrix: #This matrix allows multiple parallel jobs to create tags in the various repos at the same time
-          repo: [3scale/porta, 3scale/apisonator, 3scale/APIcast, 3scale/3scale_toolbox, 3scale/zync, 3scale/apicast-operator, 3scale/3scale-operator]
-      name: Create Release for ${{ matrix.repo }}
+          repo: [3scale/porta, 3scale/apisonator, 3scale/APIcast, 3scale/zync, 3scale/apicast-operator, 3scale/3scale-operator]
       steps:
+        - name: Create Release for ${{ matrix.repo }}
         - uses: aurelien-baudet/workflow-dispatch@v2.1.1
```

is this a **new** YAML array item, or part of the same one that the ` name: Create Release for ${{ matrix.repo }}` property belongs to? if so, probably the extra `-` below is not needed:

```suggestion
          uses: aurelien-baudet/workflow-dispatch@v2.1.1
```